### ISSUE TITLE
greetings from gnukhata team

### ISSUE THREAD

**Issue by test2a** [Mittal-Analytics/gst-e-invoicing#6]
hi.
I am one of the contributors to Gnukhata https://gitlab.com/gnukhata/gkapp/ and i came across your repo and long story short, our project is in serious need of some GST goodness and your project looks like a nice drop-in enhancement.

Your license is Gpl 3, same as ours so we've been thinking if we could use your code and not have to reinvent the wheel when you've already done most of the work.

May i (my personal question here) know why did you release this for public? i saw your website and it looks like you are into financial statement analysis and stuff. Why would you spend time to build your own GST e-invoicing library and then open sourcing it? 

That said, our GST returns, e-way bill and others are not ready for prime time because we are operating without any docs, currently we do blackbox testing by making same entries in our software and tally/marg and comparing return outputs. Haha. And we don't have e-way bill/e-invoicing so we cannot just tell people to drop their proprietary softwares and use our free one when it is not ready.

What do you think about this?

**Comment by pratyushmittal** [comment]
Hi @test2a, sure, feel free to use the project.

We would love to have contributions from your team if they make any improvements. I will review the pull-requests.

Regarding the purpose, we had to implement e-invoicing in our project. We didn't find any existing library and hence created this.

We kept it open-source because:
1. We have benefitted hugely from open-source projects. We use open-source softwares every day.
2. GST, taxation and cryptography is little hard. It is good to have more people look at the source.

Hope this helps.

Regards,
Pratyush
